css/css-writing-modes-3/table-cell-001.html

```diff
@@ -0,0 +1,42 @@
+<!doctype html>
+<html lang="en">
+<meta charset="utf-8">
+<title>CSS Writing Mode test: orthogonal table cell</title>
+<link rel="help" href="https://drafts.csswg.org/css-writing-modes-3/#block-flow">
+<meta assert="The writing mode property applies to table cells: two adjacent cells can have orthogonal writing modes, and the enclosing table must be laid out correctly.">
+<meta flags="ahem">
+<link rel="match" href="reference/table-cell-001-ref.html">
+<link rel="author" title="Florian Rivoal" href="http://florian.rivoal.net/">
+
+<style>
+table, td {
+  padding: 0;
+  border-spacing: 0;
+  font: 20px/20px ahem;
+}
+td { background: green; }
+#t2 { writing-mode: vertical-rl; }
+
+#red {
+  font: 20px/20px ahem;
+  position: absolute;
+  z-index: -1;
+  background: red;
+  height: 4em;
+  width: 5em;
+}
+
+
+</style>
+
+<p>There should be a <strong>green recangle</strong> and <strong>no red</strong>.
+
+<div id=red></div>
+
+<table><td id=t1>&nbsp;&nbsp;&nbsp;&nbsp;</td><td id=t2>&nbsp;&nbsp;&nbsp;&nbsp;</td></table>
```

Comment:
better to use a &lt;tr> here rather than testing parser error handling

Reply:
Technically the parser also adds <tbody> though if you want to go there